src/firebase.util.js

```diff
@@ -0,0 +1,35 @@
+import { initializeApp } from "firebase/app";
+import { getAuth, signInWithPopup, GoogleAuthProvider } from "firebase/auth";
+import { getDatabase, ref, set } from "firebase/database";
+import { nanoid } from "nanoid"
+
+const firebaseConfig = {
+    apiKey: "AIzaSyAJMsl-DUS_1ZUYkHWo-7ecaZ_udUm_h2o",
+    authDomain: "polling-sys.firebaseapp.com",
+    databaseURL: "https://polling-sys-default-rtdb.firebaseio.com",
+    projectId: "polling-sys",
+    storageBucket: "polling-sys.appspot.com",
+    messagingSenderId: "518429794804",
+    appId: "1:518429794804:web:aabb01591424938a7759a1"
+  };
+  
+const firebaseApp = initializeApp(firebaseConfig);
+const provider = new GoogleAuthProvider();
+  
+provider.setCustomParameters({   
+    prompt : "select_account "
```

Comment:
**Suggestion:** Remove the trailing space in the `prompt` parameter to ensure consistent behavior across browsers. [possible bug, importance: 8]
```suggestion
    prompt: "select_account"
```